parsec/core/gui/organization_info_widget.py

```diff
@@ -88,6 +88,17 @@ def __init__(
                         limit=config.active_users_limit
                     )
                 )
+            self.label_sequestration_state.setToolTip(
+                translate("TEXT_ORG_INFO_SEQUESTRATION_TOOLTIP")
+            )
+            if getattr(config, "sequester_authority", None):
+                self.label_sequestration_state.setText(
+                    translate("TEXT_ORG_INFO_SEQUESTRATION_ACTIVATED")
+                )
+            else:
+                self.label_sequestration_state.setText(
```

Comment:
Would be better not to say anything about sequester if it is not activated: otherwise the user might think the sequester can get activated in the future

Reply:
Label is now hidden in the sequestration is not activated.